### PR TITLE
Remove UnknownEthereumToken

### LINKED ISSUE #1752: Think on how UnknownEthereumToken should be handled when it becomes known
## Abstract

This is very closely related to how we should handle custom token additions https://github.com/rotki/rotki/issues/1519

At the moment unknown tokens from uniswap are treated as `UnknownEthereumToken` with an address a symbol and decimals.
But at some point support for them may be added and we have to be able to handle that.

Also it's quite possible that the idea of custom tokens as per #1519 and the UnknownEthereumToken should just be merged with this.

At the moment `UnknownEthereumToken` tries to closely resemble the `Asset` interface

## Task

Think how to handle this

### PR DESCRIPTION
**Pull request by LefterisJP**
Fix #1752

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/aave/graph.py [hunk 1]
```diff
@@ -60,23 +60,6 @@
...
-      id
-    }
-    reserve {
-      id
-    }
-    timestamp
-  }
-}
-"""
-
```
Comment (yabirgb) [inline rotkehlchen/chain/ethereum/modules/aave/graph.py:79]: Why did you remove this?

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/aave/graph.py:79]: not used anywhere
